drivers/storage/nvme/nvme.inf

```diff
@@ -0,0 +1,80 @@
+; Copyright (c) 2002-2012 ROS group
+
+[Version]
+Signature="$Windows NT$"
+Provider=%ROS%
+ClassGUID={4D36E97D-E325-11CE-BFC1-08002BE10318}
+Class=System
+DriverVer=08/11/2023, 0.0.1
+
+[DestinationDirs]
+DefaultDestDir = 11
+
+[ControlFlags]
+ExcludeFromSelect = *
+
+[Manufacturer]
+%ROS%=NVME_COMM
+
+[NVME_COMM]
+%NVME_MGMT_PORT.DeviceDesc%=nvme_inst, PCI\CC_0108
```

Comment:
Inconsistent hardware id with txtsetup.sif entry, should be fixed here or there.

Reply:
Oops good catch. Will fix